index.html

```diff
@@ -1232,14 +1232,15 @@ <h2 id="automation">
       <div class="algorithm">
         <p>
           To <dfn data-for="WebDriver">set a permission</dfn> given a {{PermissionDescriptor}}
-          |descriptor:PermissionDescriptor|, and a {{PermissionState}} |state:PermissionState|:
+          |descriptor:PermissionDescriptor|, a {{PermissionState}} |state:PermissionState|, and an
+          |origin|:
```

Comment:
Let's change this to "and an optional origin", then we don't have to pass in null down on line 1328.

Reply:
Makes sense, changed it.